src/js/popup.js

```diff
@@ -397,7 +397,7 @@ function share() {
   }
 
   if (tracking.length) {
-    shareMessage += "\n\n" + chrome.i18n.getMessage("share_tracker_header", [tracking.length, POPUP_DATA.tabUrl]) + ":\n\n";
+    shareMessage += "\n\n" + chrome.i18n.getMessage("share_tracker_header", [tracking.length, window.extractHostFromURL(POPUP_DATA.tabUrl)]) + ":\n\n";
```

Comment:
You should use `POPUP_DATA.tabHost` instead.

Reply:
And we should make the colon part of the localized string (to better support right-to-left languages).